pkg/frontend/v2/frontend_scheduler_worker.go

```diff
@@ -68,10 +68,10 @@ func newFrontendSchedulerWorkers(cfg Config, frontendAddress string, requestsCh
 			Name: "cortex_query_frontend_workers_enqueued_requests_total",
```

Comment:
Should we remove or deprecate this metric? We'll be able to get the same information at with `cortex_query_frontend_enqueue_duration_seconds` now.

Reply:
Separate PR, see: https://github.com/grafana/mimir/pull/6121. We offer no guarantee on metrics, so no need to deprecate: we can just remove it (moreover it's not used in any of the dashboards / alerts we provide).